plugins/outputs/cloudwatch/aggregator_test.go

```diff
@@ -60,6 +62,8 @@ func TestAggregator_NotDurationType(t *testing.T) {
 
 	assertNoMetricsInChan(t, metricChan)
 	close(shutdownChan)
+	// Cleanup
+	wg.Wait()
```

Comment:
Does this need to be addressed now or in other PR and I don't see any wg.Add(1) or wg.Done() though?

Reply:
Yes, it was needed due to intermittently failing tests.